mono/tests/rgctx-thread-static.cs

```diff
@@ -0,0 +1,30 @@
+using System;
```

Comment:
Would be better to add this one of the jit test files like generics.cs.

Reply:
I tried it but the issue doesn't reproduce there due to class loading order I think